### PR TITLE
test pr #5098 and merge confilct

### PR DESCRIPTION
**Pull request by squirrelChen**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -19,3 +19,15 @@
...
+[submodule "cocos2dx/platform/third_party/winrt/libcurl"]
+	path = cocos2dx/platform/third_party/winrt/libcurl
+	url = https://github.com/martell/libcurl-winrt.git
+[submodule "cocos2dx/platform/third_party/winrt/winrtsock"]
+	path = cocos2dx/platform/third_party/winrt/winrtsock
+	url = https://github.com/martell/winrtsock.git
+[submodule "cocos2dx/platform/third_party/winrt/libwebsockets"]
+	path = cocos2dx/platform/third_party/winrt/libwebsockets
+	url = https://github.com/martell/libwebsockets.git
+[submodule "cocos2dx/platform/third_party/winrt/openssl"]
```
Comment (dumganhar) [inline .gitmodules]: Do you use the source codes for these third party libraries?
If so, we need to build it to static libraries.

Comment (martell) [inline .gitmodules]: Hey dumganhar,
The third_party libraries are done the same way dale stamen has done the angle project.
We should at the end make static libs for all of them. For now I just matched his structure.

**Comment by martell** [comment]
@dumganhar koowolf has removed the submodules here and added them as static libs.
I didn't notice that previously. Could you merge this PR please?
Dale said to me in another thread that he will do the same for libangle and the other specific wp8 libraries during the week.

**Comment by squirrelChen** [comment]
@martell I used them as static libraries, but it takes a lot of time to wait while generaling code. I think had better to use dynamic libraries, just wait for @stammen to submit.

**Pull request closed without merging**